blueprint/blueprint-spring/pom.xml

```diff
@@ -119,13 +119,13 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
-            <version>4.2.2.RELEASE</version>
+            <version>6.1.15</version>
```

Comment:
I checked that spring 6.x does not work with minimal java 8 that we should have in the repo
you can find the errors by running:
```
mvn -f blueprint/blueprint-spring clean package
```
or checking your branch with definition of blueprint build from https://github.com/apache/aries/pull/297

Reply:
yes, spring 6 is for java 17, so if  java 8 support is required, the latest version is 5.3.39.